desktop-app/package.json

```diff
@@ -121,17 +121,24 @@
     "electron-updater": "^6.1.8",
     "emitter": "^0.0.5",
     "javascript-time-ago": "^2.5.10",
+    "lodash": "^4.17.21",
     "mousetrap": "^1.6.5",
     "os": "^0.1.2",
     "postcss": "^8.4.31",
     "re-resizable": "^6.9.9",
-    "react": "^18.2.0",
+    "react": "^18.3.1",
     "react-detect-click-outside": "^1.1.7",
     "react-dnd": "^16.0.1",
     "react-dnd-html5-backend": "^16.0.1",
     "react-dom": "^18.2.0",
+    "react-grid-layout": "^1.4.4",
     "react-markdown": "^8.0.7",
+    "react-masonry-component": "^6.3.0",
+    "react-masonry-css": "^1.0.16",
+    "react-masonry-list": "^1.0.6",
+    "react-measure": "^2.5.2",
```

Comment:
@manojVivek we are adding a lot of new dependencies here from a third party, do we want this? this same result should be achievable without so many dependencies

Reply:
I don't feel strongly against it, but please go ahead and add your suggestion if you feel this is unnecessary.